design/ui/pagination/src/Pagination.test.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
create-react-app does a good job of always importing this extend-expect. It would be nice if we also had that in the future.